roles/chrony/tasks/main.yml

```diff
@@ -5,6 +5,14 @@
 
 - name: Restart chrony
   become: true
-  tags: [chrony,chrony-config,chrony-restart]
+  tags: [chrony, chrony-config, chrony-restart]
   command: /bin/true
   notify: Restart chrony
+  when: ansible_pkg_mgr == 'apt'
```

Comment:
You don't need to do this. The task to restart chrony should be only one, and it doesn't use a package manager.

Reply:
I resolved this problem in my last commit.